.gitignore

```diff
@@ -1,7 +1,6 @@
 pkg/
 Gemfile.lock
 Gemfile.local
-vendor/
```

Comment:
@KosherSalt https://tickets.puppetlabs.com/browse/MODULES-3042 is for your issue.